### ISSUE TITLE
Integrate with existing 3D software

### ISSUE THREAD

**Issue by XuequanZhang** [xamarin/urho#378]
Hello，I have written a 3D software using SharpDX, but the simulation ability is limited. I want to integrate the Urho engine, but it doesn't cause too much changes to the original system.  How to render Urho scene and the original scene in the same control. How to integrate Urho as a basic rendering engine?
Anyone have an idea? Thanks very much!

**Comment by utekai** [comment]
Yes, start with the samples. They will guide you initially to where you can go.

You can work with urhosharp or with urho3d directly, depending upon your target devices.
